data/assets/scene/solarsystem/missions/artemis/actions.asset

```diff
@@ -0,0 +1,43 @@
+local setup_launch = {
+  Identifier = "os.missoins.artemis.setup.launch",
+  Name = "Set to Artemis-1 launch time",
+  Command = [[
+    openspace.time.setTime('2022-11-16T08:19:00.000');
+  ]],
+  Documentation = "Set the time to the launch time of Artemis-1",
+  GuiPath = "/Artemis",
+  IsLocal = false
+}
+
+local flyto_artemis = {
```

Comment:
Do we need this action since it just replicates what we have in the menu already?